roles/dashd/templates/dash.conf.j2

```diff
@@ -65,3 +65,7 @@ addnode={{ dashd_seednode }}:{{ dashd_port }}
 masternode=1
 masternodeprivkey={{ masternode.privkey }}
 {% endif %}
+
+{% if dashd_sporkkeyid is defined %}
+sporkkeyid={{ dashd_sporkkeyid }}
```

Comment:
Note: Might need to update this arg depending on result of `sporkkeyid` discussion in Core PR (e.g. sporkaddr or whatever is settled on).